packages/adapters/wagmi/src/client.ts

```diff
@@ -647,7 +645,6 @@ export class WagmiAdapter implements ChainAdapter {
           }
         } else if (status === 'connected' && address && chainId) {
           const caipAddress = `eip155:${chainId}:${address}` as CaipAddress
-          this.appKit?.resetAccount(this.chainNamespace)
```

Comment:
If we've connected, we don't have to resetAccount. This should have been removed long ago - I remember we removed these